Devtools: puppeteer-firefox is not quite optional yet

**Environment:**
 - **WebdriverIO version:** 5.18.6
 - **Mode:** WDIO Testrunner
 - **If WDIO Testrunner, running sync/async:** sync
 - **Node.js version:** 12.8.1
 - **NPM version:** 6.10.2
 - **Browser name and version:** Chrome 79
 - **Platform name and version:** Windows 10
 - **Additional wdio packages used (if applicable):** Defaults installed as part of getting started guide

**Config of WebdriverIO**
Add following line to default auto-generated config:
`automationProtocol: 'devtools',`

**Describe the bug**
Test runs fail when using `automationProtocol: 'devtools'` if devtools package was installed with` --no-optional` flag.

**To Reproduce**
Steps to reproduce the behavior:
- follow [getting started guide](https://webdriver.io/docs/gettingstarted.html) 
- install devtools with --no-optional flag `npm install --save devtools --no-optional`
- add `automationProtocol: 'devtools'` to wdio.config.js file
- Unexpected error: `Error: Cannot find module 'puppeteer-firefox'`

**Expected behavior**
Test run should not fail due to error trying to resolve the puppeteer-firefox module.

**Additional context**
Original issue for making puppeteer-firefox an optional dependency: #4481

--- a/packages/devtools/src/commands/status.js
+++ b/packages/devtools/src/commands/status.js
@@ -2,8 +2,23 @@ import path from 'path'
 
 const puppeteerPath = require.resolve('puppeteer-core')
 const puppeteerPkg = require(`${path.dirname(puppeteerPath)}/package.json`)
-const puppeteerFirefoxPath = require.resolve('puppeteer-firefox')
-const puppeteerFirefoxPkg = require(`${path.dirname(puppeteerFirefoxPath)}/package.json`)
+
+let puppeteerFirefoxPath
+let puppeteerFirefoxPkg
+
+try {
+    puppeteerFirefoxPath = require.resolve('puppeteer-firefox')
+} catch (error) {
+    /*
+     * puppeteer-firefox is an optional dependency and we expect to ignore
+     * an error from require.resolve if it's not installed
+     */
+}
+
+// Only require puppeteer-firefox package.json if it's installed
+puppeteerFirefoxPkg = (puppeteerFirefoxPath) ?
+    require(`${path.dirname(puppeteerFirefoxPath)}/package.json`) :
+    {}
 
 export default async function status () {
     return {

